http/src/main/resources/reference.conf

```diff
@@ -392,7 +392,7 @@ zombieRuntimeMonitor {
 }
 
 persistentDisk {
-  defaultDiskSizeGB = 500
+  defaultDiskSizeGB = 20
```

Comment:
default size for persistent disk

Reply:
It may be best to specify a bit higher of a default. I think individual files users often stream/interact with can be around 20GB

Reply:
are you thinking 20MB or 20GB for individual file?

Reply:
20GB. When doing the DRS work recently, a common use case a user told me about to test was copying over 20GB .cram files. In general, files containing genetic data can be very large

Reply:
do you they usually specify disk size in this case? the current disk size in total is 50G for gce VMs...

Reply:
They can always specify a size. I'm just thinking, in a typical exploratory analysis use-case, 20GB may be a bit small since individual files in the genomic space could exceed this size.